content/02.main-text.md

```diff
@@ -246,20 +251,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-In our open review paper, we established [contributor guidelines](https://github.com/greenelab/deep-review/blob/v0.9-preprint/CONTRIBUTING.md) that discussed norms in the areas of text contribution, peer review, and authorship, which we identified in advance as potential areas of disagreement.
+In the Deep Review, we established [contributor guidelines](https://github.com/greenelab/deep-review/blob/v0.9-preprint/CONTRIBUTING.md) that discussed norms in the areas of text contribution, peer review, and authorship, which we identified in advance as potential areas of disagreement.
 Our contributor guidelines required verifiable participation: either directly attributable changes to the text or participation in the discussion on GitHub.
-We maintained our guidelines, even when a case arose where two authors had written text together but only one had directly attributable changes and participation.
```

Comment:
To my recollection, this was relevant to the initial Deep Review version but not the final one.